packages/gatsby-theme-newrelic/src/components/ContributingGuidelines.js

```diff
@@ -95,6 +104,8 @@ const ContributingGuidelines = ({ fileRelativePath }) => {
 
 ContributingGuidelines.propTypes = {
   fileRelativePath: PropTypes.string,
+  title: PropTypes.string,
```

Comment:
What about naming this `issueTitle`? In the context of this component, `title` makes it feel like the name of the section, when really this is for titling the issue itself. This feedback would also apply tot he `GlobalFooter`

Reply:
How about `pageTitle`, since that is what this prop is supposed to represent.

Reply:
You're right! I like that better. Go with that!